kie-internal/src/main/java/org/kie/internal/persistence/jpa/JPAKnowledgeService.java

```diff
@@ -162,7 +162,8 @@ private static synchronized KieStoreServices getJPAKnowledgeServiceProvider() {
     private static void loadProvider() {
         try {
             // we didn't find anything in properties so lets try and us reflection
```

Comment:
Is this comment this accurate? I can't see any code that would be looking for the info in properties.